src/components/card/Card.jsx

```diff
@@ -58,27 +58,41 @@ class Card extends Component {
   }
 
   getAssignees() {
-    if (this.props.assignees.length > 2) {
+    if (this.props.assignees.length > 3) {
       const members = this.props.assignees
+      if (this.props.cardResponsible === {}) {
+        // orders member by alphabetical order
+        members.sort((a, b) => a.initials < b.initials)
+      } else {
+        // if responsible is in members, sets responsible to be the firs user
+        if (members.find(element => element.id === this.props.cardResponsible.id) === true) {
```

Comment:
You don't need `=== true`, the boolean checking is implicit with a `if`.

Reply:
Doesn't work without true, the condition will always be met and we'll face errors when responsible isn't in members. I've tested it.